services/redis-commander/docker-compose.yml

```diff
@@ -23,9 +23,11 @@ services:
         - traefik.http.routers.redis.middlewares=ops_auth@docker, ops_gzip@docker, ops_whitelist_ips@docker
       resources:
         limits:
-          memory: 128M
+          memory: 128M  # periodically hits this limit dalco-prod. Should we increase?
```

Comment:
Let's increase to 192M or 256M then. Redis commander is actually used by the developers regularly. :--)

Reply:
I have increased to 192M